src/components/map/MapComponent.vue

```diff
@@ -121,11 +120,14 @@ async function updateViewPoint(viewPoint: string) {
       'nom',
       stationsStore.currentStationView!
     )
-    let viewpoint: Viewpoint | null = getViewpointFromFeature(featureStation)
-    if (viewpoint !== null) {
-      const newVp = cloneViewPointAndResetCameraPosition(viewpoint, null)
-      await activeMap.gotoViewpoint(newVp)
+    let viewpoint: Viewpoint | null = getViewpointFromFeatureDistance(
+      featureStation,
+      2000
```

Comment:
Why set a distance to 2000 here ?

Reply:
the distance where the station looks good. I added a comment for future reference.